pkg/azure/test_utils.go

```diff
@@ -428,13 +449,32 @@ func getFakeSubnet() *armnetwork.Subnet {
 
 func getFakeVirtualNetwork() *armnetwork.VirtualNetwork {
 	return &armnetwork.VirtualNetwork{
-		Name:     to.Ptr(validVnetName),
+		Name:     to.Ptr(validParagliderVnetName),
 		ID:       to.Ptr(validVnetId),
 		Location: to.Ptr(testLocation),
 		Properties: &armnetwork.VirtualNetworkPropertiesFormat{
 			AddressSpace: &armnetwork.AddressSpace{
 				AddressPrefixes: []*string{to.Ptr(validAddressSpace)},
 			},
+			Subnets: []*armnetwork.Subnet{
+				{
+					Name: to.Ptr(validSubnetName),
+					ID:   to.Ptr(validParagliderSubnetId),
+				},
+			},
+		},
+	}
+}
+
+func getFakeVirtualNetworkWithUnusedAddressSpace() *armnetwork.VirtualNetwork {
```

Comment:
I assume this is for creating fake vnets for resources to attach? I think a better name might make that more clear. How about something like `getFakeUnattachedVirtualNetwork` (and then you could follow this naming convention for other relevant things if necessary.

Reply:
Sounds good. Will fix this